packages/core/src/Stores/Helpers/client-notifications.js

```diff
@@ -62,6 +62,17 @@ export const getCashierValidations = cashier_arr => {
 };
 
 // Notifications keys will not be added to localStorage and will appear again after user logout/login
-export const excluded_notifications = ['contract_sold', 'switched_to_real', 'has_changed_two_fa'];
+export const excluded_notifications = [
+    'contract_sold',
+    'switched_to_real',
```

Comment:
@aizad-deriv in previous PR you deletes switched_to_real from this array, here you left it :) what is right?

Reply:
Already removed 😄 👍🏼